docs/review-checklist.md

```diff
@@ -1,3 +1,63 @@
+
+# Code style
+
+Code should conform to the project's coding style. We use Seastar's
+coding style, see seastar/coding-style.md.
+
+# Naming
```

Comment:
I think this should be in coding-style.md.
If it isn't, coding-style.md should be removed, and its content incorporated here. We don't need these two files fighting for the honor of documenting coding standards.